mailpoet/assets/js/src/newsletters/listings/re_engagement.jsx

```diff
@@ -249,22 +249,31 @@ class NewsletterListReEngagementComponent extends Component {
       (match, i) => <SegmentTags segments={newsletter.segments} key={i} />,
     );
 
-    let frequencyKey = 'reEngagementFrequencyMonth';
+    let frequency = _x(
```

Comment:
I have noticed @samnajian  that you didn't use the third parameter `domain` here.
It is not used also in other calls in this commit.

Reply:
ops! I had done a double check for other commits, but copy/pasted these from template files and missed adding the domain! thanks for the catch. Add them in fc504c2ac95b28a4ff39b93751b80c8bcddd02e5